src/pal/src/misc/sysinfo.cpp

```diff
@@ -384,6 +384,53 @@ PAL_HasGetCurrentProcessorNumber()
     return HAVE_SCHED_GETCPU;
 }
 
+bool
+PALAPI
+PAL_ReadMemoryValueFromFile(const char* filename, size_t* val)
```

Comment:
@janvorli This is a cut/paste rename from `CGroup::ReadMemoryValueFromFile() `  Seems wrong to create a third clone of this function.  Whare should it go.